src/seedu/addressbook/AddressBook.java

```diff
@@ -202,13 +211,20 @@ public static void main(String[] args) {
         showWelcomeMessage();
         processProgramArgs(args);
         loadDataFromStorage();
-        while (true) {
+        startUserCommandFeedbackLoop();
+    }
+
+	/**
+	 * Starts a user input, console output loop for the user
+	 */
+	private static void startUserCommandFeedbackLoop() {
+		while (true) {
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.